apps/rush-lib/src/utilities/Utilities.ts

```diff
@@ -596,21 +690,20 @@ export class Utilities {
     // into node-core-library, but for now this hack will unblock people:
 
     // Only escape the command if it actually contains spaces:
-    const escapedCommand: string = command.indexOf(' ') < 0 ? command
+    const escapedCommand: string = command.indexOf(' ') < 0
+      ? command
       : Utilities.escapeShellParameter(command);
 
     const escapedArgs: string[] = args.map((x) => Utilities.escapeShellParameter(x));
 
     let result: child_process.SpawnSyncReturns<Buffer> = child_process.spawnSync(escapedCommand,
       escapedArgs, options);
 
-    /* tslint:disable:no-any */
-    if (result.error && (result.error as any).errno === 'ENOENT') {
+    if (result.error && (result.error as any).errno === 'ENOENT') { // tslint:disable-line:no-any
       // This is a workaround for GitHub issue #25330
       // https://github.com/nodejs/node-v0.x-archive/issues/25330
```

Comment:
Out of scope for this PR, but man we should really replace this logic with [Executable.spawnSync](https://github.com/Microsoft/web-build-tools/blob/577efd3d2251e72649aa8f7b7a475ea7b77132a2/libraries/node-core-library/src/Executable.ts#L142).  It's a much more complete solution to this problem.